src/CMakeLists.txt

```diff
@@ -223,7 +223,8 @@ if(build_common)
         set(MOOS_LIBRARY_HEADER "MOOSLIB/MOOSLib.h" PARENT_SCOPE)
       endif()
 
-      if(MOOS_FOUND)
+      find_package(Proj)
```

Comment:
I think this should be ```find_package(Proj QUIET)``` so that it fails through quietly (setting build_moos=OFF) when users build without MOOS (and proj).